cookbook/core/type_system/pytorch_types.py

```diff
@@ -0,0 +1,192 @@
+"""
+.. _pytorch_types:
+
+PyTorch Types
+=============
+
+Flyte promotes the use of strongly-typed data to make it easier to write pipelines that are more robust and easier to test.
+Flyte is primarily used for machine learning besides data engineering. To simplify the communication between Flyte tasks, especially when passing
+around tensors and models, we added support for the PyTorch types.
+"""
+
+# %%
+# Tensors & Modules
+# -----------------
+#
+# Many a times, we need to pass around tensors and modules (aka models). In the absence of native type support for PyTorch tensors and modules,
```

Comment:
When you say "we need to", you mean user? If so, can we change to "when you need to pass ..."